ibis/backends/mssql/datatypes.py

```diff
@@ -31,14 +31,14 @@ def _type_from_result_set_info(col: _FieldDescription) -> dt.DataType:
             typ = dt.Float32
         else:
             typ = dt.Float64
-    return typ(nullable=True)
+    return typ(nullable=col["is_nullable"])
```

Comment:
mysql backend has always nullable parameters set to `True`.
https://github.com/ibis-project/ibis/blob/0415bd3db3777a2c29793703a6fc3764e097255a/ibis/backends/mysql/datatypes.py#L65-L66

Reply:
I don't think so. For one, in the `mssql` backend, the oddball `TIMESTAMP` type seems to have `is_nullable` set to `False`, which is why I changed this.